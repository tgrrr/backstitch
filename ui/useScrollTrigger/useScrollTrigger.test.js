import React from 'react'
import renderer from 'react-test-renderer'
import { useScrollTrigger } from '.'

it('useScrollTrigger: default', () => {
    const component = renderer.create(
        <useScrollTrigger>
            <>
                useScrollTrigger
            </>
        </useScrollTrigger>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
