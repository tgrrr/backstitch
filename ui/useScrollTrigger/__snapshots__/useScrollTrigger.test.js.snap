// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useScrollTrigger: default 1`] = `
<useScrollTrigger>
  useScrollTrigger
</useScrollTrigger>
`;
